assets/js/state/sagas/sapSystems.js

```diff
@@ -62,6 +64,22 @@ function* applicationInstanceRegistered({ payload }) {
   );
 }
 
+export function* applicationInstanceDeregistered({ payload }) {
+  yield put(removeApplicationInstance(payload));
+  yield put(
+    appendEntryToLiveFeed({
+      source: payload.sid,
+      message: 'Application instance deregistered.',
+    })
+  );
+  yield put(
+    notify({
+      text: `An application instance has been deregistered from ${payload.sid}.`,
```

Comment:
The same as for the database `The application instance ${instance_number} has been...`